x-pack/legacy/plugins/apm/server/lib/apm_telemetry/index.ts

```diff
@@ -4,9 +4,57 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-export {
-  storeApmTelemetry,
-  createApmTelementry,
-  APM_TELEMETRY_DOC_ID
-} from './apm_telemetry';
-export { makeApmUsageCollector } from './make_apm_usage_collector';
+import { Server } from 'hapi';
+import { countBy } from 'lodash';
+import { SavedObjectAttributes } from 'src/core/server';
+import { InternalCoreSetup } from 'src/core/server';
+import { isAgentName } from '../../../common/agent_name';
+import { getSavedObjectsClient } from '../helpers/saved_objects_client';
+export const APM_TELEMETRY_DOC_ID = 'apm-telemetry';
+
+export function createApmTelementry(
+  agentNames: string[] = []
+): SavedObjectAttributes {
+  const validAgentNames = agentNames.filter(isAgentName);
+  return {
+    has_any_services: validAgentNames.length > 0,
+    services_per_agent: countBy(validAgentNames)
+  };
+}
+
+export async function storeApmTelemetry(
```

Comment:
Can you rename this, so it's clear that it's just telemetry for the service overview page? Something like `storeApmServicesTelemetry`